syntax = "proto3";

package ecommerce;

service ProductInfo{
  rpc addProduct(Product) returns (ProductId);
  rpc getProduct(ProductId) returns (Product);
}

message ProductId{
  int32 value = 1;
}

message Product{
  string id = 1;
  string name = 2;
  string description =3;
  float price = 4;
}
